modules/instance-template/variables.tf

```diff
@@ -73,6 +73,12 @@ variable "image_family" {
   default     = "cos-stable"
 }
 
+variable image_name {
+  description = "The GCE container-optimized image to run on the instance."
+  type        = string
+  default     = ""
```

Comment:
Please make sure that "terraform-google-modules/container-vm/google" module works as expected with this default value.